src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
-
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        System.out.println(firstBall);
+        Ball secondBall = Lottery.getRandomBall();
+        System.out.println(secondBall);
+        Ball thirdBall = Lottery.getRandomBall();
```

Comment:
Don't use static methods in your solution. Instead of calling `Lottery.getRandomBall()` statically, you should create an instance of the `Lottery` class and use it to get the random balls. This will make your code more object-oriented and easier to manage.